assistant_dialog_skill_analysis/utils/lang_utils.py

```diff
@@ -33,67 +36,59 @@ def tokenize(self, sentences):
     def init_resources(self):
         self.punctuation_pattern = re.compile("|".join(PUNCTUATION))
         self.stemmer = None
+        resource_path = os.path.join(
```

Comment:
I wouldn't call it `resource_path` because it's just path to `stopwords`. It would be `resource_path` without the final `stopwords`. This way, `stopwords_path` would be a better name I think